ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,530 @@
+This works with projects and proxied maintainers alike.
+</p>
+
+<note>
+For Gentoo developers, the developer tag is enough, i.e.,<c>
+;maintainer=developer</c>
+</note>
+
+</body>
+</subsection>
+
+<subsection>
+<title>Daily / Occassional pkgcheck scans</title>
+<body>
+
+<p>
+<c>pkgcheck</c> provides a tool to quickly check the overall state of your
+packages. To check for pending stabilizations, you can use something like:
+<c>git grep -l developer@gentoo.org '**/metadata.xml' | cut -d/ -f1-2 | xargs
+pkgcheck --color true scan -c StableRequestCheck</c>
+in the root of the git tree.
+</p>
+
+<p>
+To see packages with identical KEYWORDS, which means you can clean some older
+versions from the tree, use: <c>git grep -l developer@gentoo.org
+'**/metadata.xml' | cut -d/ -f1-2 | xargs pkgcheck --color true scan -c
+RedundantVersionCheck</c>
+</p>
+
+<p>
+and finally, to combine these checks, you can do:
+<c>git grep -l developer@gentoo.org '**/metadata.xml' | cut -d/ -f1-2 | xargs
+pkgcheck --color true scan -c RedundantVersionCheck,StableRequestCheck</c>
+</p>
+
+<p>
+These two above checks are also shown in your maintainer page on <uri
+link="https://packages.gentoo.org/">packages.gentoo.org</uri>.
+</p>
+
+<p>
+Finally, check available <e>PYTHON_COMPAT</e> updates to your packages with:
+
+<c>git grep -l developer@gentoo.org '**/metadata.xml' | cut -d/ -f1-2 | xargs
+pkgcheck --color true scan -k PythonCompatUpdate</c>
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>External resources</title>
+<body>
+<ul>
+  <li>
+    <uri link="https://blogs.gentoo.org/mgorny/2019/12/12/a-better-ebuild-workflow-with-pure-git-and-pkgcheck/">
+      A better ebuild workflow with pure git and pkgcheck
+    </uri>
+  </li>
+  <li>
+    <uri link="https://wiki.gentoo.org/wiki/Project:Infrastructure/Generating_GLEP_63_based_OpenPGP_keys">
+      Generating GLEP 63 based OpenPGPkeys
+    </uri>
+  </li>
+  <li>
+    <uri link="https://wiki.gentoo.org/wiki/Gentoo_GitHub#How_to_make_a_pull_request">
+      Gentoo GitHub - How to make a pull request
+    </uri>
+  </li>
+  <li>
+    <uri link="https://dev.gentoo.org/~mgorny/python-guide/">
+      Gentoo Python's guide
+    </uri>
+  </li>
+  <li>
+    <uri link="https://projects.gentoo.org/qa/policy-guide/">
+      Gentoo QA's Policy Guide
+    </uri>
+  </li>
+  <li>
+    <uri link="https://wiki.gentoo.org/wiki/Package_testing">
+      Package testing
+    </uri>
+  </li>
+  <li>
+    <uri link="https://qa-reports.gentoo.org/">QA reports</uri>
+  </li>
+  <li>
+    <uri link="https://repology.org">Repology</uri>
+  </li>
+  <li>
+    <uri link="https://wiki.gentoo.org/wiki/Test_environment">
+      Test environment
+    </uri>
+  </li>
+  <li>
+    <uri link="https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html">
+      Ultimate guide to EAPI-7
+    </uri>
+  </li>
+  <li>
+    <uri link="https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html">
+      Ultimate guide to EAPI-8
+    </uri>
+  </li>
```

Comment:
The relevant parts of that guide are now in the EAPI 8 section (and there you'll find the link as well), so no need to link it from here as well.

In general, I see this whole large collection of links somewhat critical. It make the chapter look like a web page from the nineties. :grimacing:

Reply:
Just the way I like it ;)

sure I'll remove the block.

Reply:
Should be all done now.